bin/check-uptime.rb

```diff
@@ -35,11 +36,18 @@ class CheckUptime < Sensu::Plugin::Check::CLI
          proc: proc(&:to_i),
          default: 180
 
+  option :reverse,
+         short: '-r',
+         long: '--reverse',
+         description: 'Reverse, Warn if uptime is upper SEC',
+         boolean: true,
+         default: false
+
   def run
     uptime_sec  = IO.read('/proc/uptime').split[0].to_i
     uptime_date = Time.now - uptime_sec
 
-    if uptime_sec < config[:warn]
+    if uptime_sec < config[:warn] || (uptime_sec > config[:warn] && config[:reverse])
```

Comment:
I think it might make more sense to use a more standard if, elsif statement like this:
```ruby
if config[:reverse_comparison] && uptime_sec > config[:warn]
  message "System boot detected (#{uptime_sec} seconds up), compared using '>'"
  warning
elsif uptime_sec < config[:warn]
  message "System boot detected (#{uptime_sec} seconds up), compared using <"
  warning
end
```

Reasoning is that it makes no sense to check less than if we know we want to check greater than. Just seems more efficient and easier to follow IMHO.